### PR TITLE
Changed attributes order in css checking functions

### PR DESCRIPTION
**Pull request by mjaskot**

### CODE REVIEW

File: cypress/integration/home/home_spec.js [hunk 1]
```diff
@@ -208,37 +208,36 @@ describe('Homepage', () => {
...
         ))
 
       it('"Clutch" section CSS styles', () =>
         cy
           .get(clutch.sectionSelector)
-          .within(() => clutch.headers.forEach(header => cy.checkCSSParameters({ containText: header }, headerStyles))))
+          .within(() => clutch.headers.forEach(header => cy.checkCSSParameters(headerStyles, { containText: header }))))
 
-      it('Checks "Contact input border CSS styles"', () =>
+      it.only('Checks "Contact input border CSS styles"', () =>
```
Comment (hypeofpipe) [inline cypress/integration/home/home_spec.js]: @mjaskot remove it please

Comment (hypeofpipe) [inline cypress/integration/home/home_spec.js]: Remove 'only' please

File: cypress/integration/about/about_spec.js [hunk 2]
```diff
@@ -86,7 +86,7 @@ describe('About us page', () => {
       ]
 
       elementsToProps.forEach(({ elements, props }) =>
-        elements.forEach(element => it(`Testing \'${element}\'`, () => cy.checkCSSParameters({ containText: element }, props)))
+        elements.forEach(element => it(`Testing \'${element}\'`, () => cy.checkCSSParameters(props, { containText: element })))
```
Comment (MykolaRudnytskyi) [inline cypress/integration/about/about_spec.js]: Width should be 80

File: cypress/integration/careers/careers_spec.js [hunk 3]
```diff
@@ -125,26 +125,26 @@ describe('Careers Page', () => {
...
         paragraphs.sectionThree.forEach(element => {
           cy.log(`Checking "${element}" CSS styles`)
-          cy.checkCSSParameters({ containText: element }, paragraphsCSS)
+          cy.checkCSSParameters(paragraphsCSS, { containText: element })
         })
 
         paragraphs.whiteText.forEach(element => {
           cy.log(`Checking "${element}" CSS styles`)
-          cy.checkCSSParameters({ containText: element }, Object.assign({}, paragraphsCSS, { color: 'rgb(255, 255, 255)' }))
+          cy.checkCSSParameters(Object.assign({}, paragraphsCSS, { color: 'rgb(255, 255, 255)' }), { containText: element })
```
Comment (MykolaRudnytskyi) [inline cypress/integration/careers/careers_spec.js]: Width should be 80

File: cypress/integration/contact/contact_spec.js [hunk 4]
```diff
@@ -249,7 +249,7 @@ describe('Contact page', () => {
           if (element.callback) {
             element.callback(element.get ? element.get : element.contains)
           }
-          cy.checkCSSParameters(element.contains ? { containText: element.contains } : { selector: element.get }, props)
+          cy.checkCSSParameters(props, element.contains ? { containText: element.contains } : { selector: element.get })
```
Comment (MykolaRudnytskyi) [inline cypress/integration/contact/contact_spec.js]: Width should be 80

File: cypress/integration/home/home_spec.js [hunk 5]
```diff
@@ -208,35 +208,34 @@ describe('Homepage', () => {
...
+              : paragraphStyles,
+            { containText: paragrah }
           )
         }))
 
       it('"7what" section CSS styles', () =>
         cy
           .get(sevenWhat.selector)
-          .within(() => sevenWhat.paragrahs.forEach(paragrah => cy.checkCSSParameters({ containText: paragrah }, paragraphStyles))))
+          .within(() => sevenWhat.paragrahs.forEach(paragrah => cy.checkCSSParameters(paragraphStyles, { containText: paragrah }))))
```
Comment (MykolaRudnytskyi) [inline cypress/integration/home/home_spec.js]: Width should be 80

**Pull request merged**